website/index.html

```diff
@@ -0,0 +1,81 @@
+		background-color: #800080;
+		background-image: none;
+		font-family: Comic Sans MS, Comic Sans;
+		font-size: 15px;
+		color: #FFFFFF;
+	}
+	
+	a:link {
+		color: #00FF00;
+	}
+	
+	body {
+		background-color: #800080;
+		background-image: none;
+		font-family: Comic Sans MS, Comic Sans;
+		font-size: 15px;
+		color: #000000;
+	}
+	
+	h1 {
+		font-size: 28px;
+		color: yellow;
+		text-align: center;
+	}
+	
+	p {
+		background-color: #FF0000;
+		background-image: none;
+		font-family: Comic Sans MS, Comic Sans;
+		font-size: 20px;
+	}
+	
+	li {
+		list-style-image: url(images/point.gif);
+	}
+</style>
+
+<body>
+	<div id="readme" class="readme blob instapaper_body">
+		<article class="markdown-body entry-content" itemprop="text">
+			<h1><a id="user-content-systemben" class="anchor" href="http://wonder-tonic.com/geocitiesizer/content.php?theme=1&url=https://github.com/benaadams/System.Ben/blob/master/README.md/#systemben" aria-hidden="true"></a>System.Ben</h1>
+			<img class="media-image" style="margin-top: 0px;" alt="Ben Adams" src="https://pbs.twimg.com/profile_images/758624302803644416/CWN6B50a.jpg" data-width="512" data-height="512">
+			<h2>"Hi! I'm Ben! Super-programmer!"</h2>
+			<p>Who says you can't be super fast and super secure</p>
+			<p>
+				<a href="http://wonder-tonic.com/geocitiesizer/content.php?theme=1&url=https://gitter.im/System-Ben/Lobby?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge&amp;utm_content=badge"><img src="https://camo.githubusercontent.com/6169107b0882bbf7e3a64445fb7217debacfc6b7/68747470733a2f2f6261646765732e6769747465722e696d2f53797374656d2d42656e2f4c6f6262792e737667"
+					alt="Join the chat at https://gitter.im/System-Ben/Lobby" data-canonical-src="https://badges.gitter.im/System-Ben/Lobby.svg"
+					style="max-width:100%;"></a>
+			</p>
+			<h2><img src="http://www.wonder-tonic.com/geocitiesizer/images/computer-01.gif"/><a id="user-content-fast" class="anchor" href="http://wonder-tonic.com/geocitiesizer/content.php?theme=1&url=https://github.com/benaadams/System.Ben/blob/master/README.md/#fast" aria-hidden="true"></a>Fast <img src="http://www.wonder-tonic.com/geocitiesizer/images/flying-horse.gif"/></h2>
+			<p>The fastest code is the code that doesn't execute; so there is no code.</p>
+			<h2><a id="user-content-secure" class="anchor" href="http://wonder-tonic.com/geocitiesizer/content.php?theme=1&url=https://github.com/benaadams/System.Ben/blob/master/README.md/#secure" aria-hidden="true"></a>Secure</h2>
+			<p>This zero api library has the smallest possible api making it very hard to attack or find vulnerabilities in</p>
+			<h2><a id="user-content-supported-frameworks" class="anchor" href="http://wonder-tonic.com/geocitiesizer/content.php?theme=1&url=https://github.com/benaadams/System.Ben/blob/master/README.md/#supported-frameworks" aria-hidden="true"></a>Supported Frameworks</h2>
+			<p>.NET Standard 1.0+ <img src="http://www.wonder-tonic.com/geocitiesizer/images/floppy-05.gif"/></p>
```

Comment:
This version is too short, please only use approved version ~~sentences~~ paragraphs from Microsoft marketing.